torch_xla/csrc/data_ops.cpp

```diff
@@ -276,14 +276,35 @@ std::vector<xla::XlaOp> BuildChunk(const torch::jit::Node* node,
   return BuildChunk(input, chunks, dim);
 }
 
-std::vector<xla::XlaOp> BuildSplit(const xla::XlaOp& input,
-                                   xla::int64 split_size, xla::int64 dim) {
+size_t ComputeSplitCount(
+    xla::int64 dim_size,
+    tensorflow::gtl::ArraySlice<const xla::int64> split_sizes) {
+  size_t count = 0;
+  for (auto size : split_sizes) {
+    if (size > dim_size) {
+      break;
+    }
+    dim_size -= size;
```

Comment:
Let's use a `remaining_dim_size` variable instead, reusing the input parameter is confusing.

Reply:
Why? I could understand if the function was 100 lines long. But this fits in 1/10 of a screen, and clearly shows that the variable is not used elsewhere.